app/services/repository_actions/repository_cell_resolver.rb

```diff
@@ -0,0 +1,113 @@
+# frozen_string_literal: true
+
+module RepositoryActions
+  class RepositoryCellResolver
+    def initialize(cell, new_row, team)
+      @cell    = cell
+      @new_row = new_row
+      @team    = team
+    end
+
+    def call
+      case @cell.value_type
+      when 'RepositoryListValue'
+        clone_repository_list_value
+      when 'RepositoryTextValue'
+        clone_repository_text_value
+      when 'RepositoryAssetValue'
+        clone_repository_asset_value
+      when 'RepositoryDateValue'
+        clone_repository_date_value
+      end
+    end
+
+    private
+
+    def clone_repository_list_value
+      old_value = @cell.value
+      RepositoryListValue.create(
```

Comment:
I think that `created_by` and `last_modified_by` should be set to the user who is performing the copying of the row. So maybe the whole `RepositoryCellResolver` needs additional parameter, `user`.

Reply:
The copied items have all columns (including hidden ones) copied from the original repository item. The only differences
are the following:
 Newly copied item is not assigned to any task, regardless of the original item’s task assignments.
 Newly copied item has a name “<original_item_name> (1)” – the (1) is appended to the end of the name.
 Newly copied item’s “added on” field is set onto the time the item has been copied.
 Newly copied item’s “added by” field is set to the user who performed the copying.
After copying is done, repository table is reloaded and a flash message appears, notifying the user that “<n> item(s)
successfully copied.” (see Figure 10: Copying repository items).

Reply:
:arrow_up: from specs

Reply:
I know this is not specified concretely in the specifications, but don't you agree it should be same as the row's `created_by`?

To me, it doesn't make sense if row's `created_by` is one thing, and then its cells - which are totally dependant on this row for existence - don't have the same `created_by` attribute. 

So please fix this nonetheless.